uisdk/src/main/java/com/karhoo/uisdk/screen/address/addresslist/AddressResultListView.kt

```diff
@@ -40,7 +41,7 @@ class AddressResultListView @JvmOverloads constructor(
         }
 
         recycler.apply {
-            layoutManager = LinearLayoutManager(context, LinearLayoutManager.VERTICAL, false)
+            layoutManager = GridLayoutManager(context, 1)
```

Comment:
We shouldn't change the LinearLayoutManager to a GridLayoutManager in order to solve a click listener not being triggered when pressing on the row.
If the layout inspector is used, we can see that the Address item view is not filling up the whole width of the screen, due to the MATCH_PARENT layout params not being applied. When inflating the view, it should be either attached to a parent so that Android knows how to measure it or set them programatically after the view is initialized

Reply:
Done